sql-plugin/src/main/320until330-all/scala/com/nvidia/spark/rapids/shims/OrcShims.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids.shims
```

Comment:
NIT:
I saw you intentionally add a space line between the header comment and the package declaration in file [OrcShims320untilAllBase.scala](https://github.com/NVIDIA/spark-rapids/pull/5066/files#diff-7c11be7692e8e80ab48e0491dd2d03e281135cc8d5075bad368ae5dbd7d7c66a).
However the space line is missing here.